components/log-viewer-webui/client/package.json

```
@@ -24,9 +24,9 @@
     "mini-css-extract-plugin": "^2.9.0",
     "react-refresh": "^0.14.2",
     "style-loader": "^4.0.0",
-    "webpack": "^5.92.1",
+    "webpack": "^5.94.0",
```

There seems to be a newer version now.

```suggestion
    "webpack": "^5.95.0",
```